src/Core/Auth/UserFeatures/WebAuthnLogin/Implementations/CreateWebAuthnLoginCredentialCommand.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Bit.Core.Auth.Entities;
+using Bit.Core.Auth.Repositories;
+using Bit.Core.Entities;
+using Bit.Core.Utilities;
+using Fido2NetLib;
+
+namespace Bit.Core.Auth.UserFeatures.WebAuthnLogin.Implementations;
+
+internal class CreateWebAuthnLoginCredentialCommand : ICreateWebAuthnLoginCredentialCommand
+{
+    public const int MaxCredentialsPerUser = 5;
+
+    private readonly IFido2 _fido2;
+    private readonly IWebAuthnCredentialRepository _webAuthnCredentialRepository;
+
+    public CreateWebAuthnLoginCredentialCommand(IFido2 fido2, IWebAuthnCredentialRepository webAuthnCredentialRepository)
+    {
+        _fido2 = fido2;
+        _webAuthnCredentialRepository = webAuthnCredentialRepository;
+    }
+
+    public async Task<bool> CreateWebAuthnLoginCredentialAsync(User user, string name, CredentialCreateOptions options, AuthenticatorAttestationRawResponse attestationResponse, bool supportsPrf, string encryptedUserKey = null, string encryptedPublicKey = null, string encryptedPrivateKey = null)
+    {
+        var existingCredentials = await _webAuthnCredentialRepository.GetManyByUserIdAsync(user.Id);
+        if (existingCredentials.Count >= 5)
```

Comment:
Shouldn't this use the class variable `MaxCredentialsPerUser`?

Reply:
Woooops, yep! Good catch!